R/tbl_hierarchical.R

```diff
@@ -99,14 +100,13 @@
 #'   by = TRTA,
 #'   overall_row = TRUE
 #' )
-#'
 tbl_hierarchical <- function(data,
                              variables,
+                             id,
+                             denominator,
                              by = NULL,
-                             id = NULL,
```

Comment:
FYI I removed the default values for arguments that are required for the users to pass.